forms-flow-web/src/components/PrivateRoute.jsx

```diff
@@ -141,7 +141,7 @@ const PrivateRoute = React.memo((props) => {
               userRoles.includes(STAFF_DESIGNER) ? (
                 <Component {...props} />
               ) : (
-                <>unauthorized</>
+                <Redirect exact to="/404" />
```

Comment:
Why is Unauthorized going to 404 ? We should be having Not Found and Unauthorized separate, right ?